lib/mutant/mutator/node/send.rb

```diff
@@ -26,8 +26,9 @@ class Send < self
           to_i:          %i[to_int],
           to_a:          %i[to_ary],
           to_h:          %i[to_hash],
-          at:            %i[fetch],
-          :[] =>         %i[at fetch],
+          at:            %i[fetch key?],
+          fetch:         %i[key?],
+          :[] =>         %i[at fetch key?],
```

Comment:
@dkubb ^^ In this particular case I trade verbosity for the time it takes to work out a "real" high level mutation strategy. Lets keep the verbosity for now and do not implement a half backed high level strategy for a specific emitter, instead later remove the explicitness when a global strategy exists.